client/src/views/Warehouse.vue

```diff
@@ -340,24 +364,20 @@ export default {
     },
   },
 
-  created() {
-    this.initialize();
+  async mounted() {
+    const response = axios.get('/api/warehouse/');
```

Comment:
Implement new changes (from ci-pipeline-update branch PR):

Add this to your <script> `const url = process.env.VUE_APP_API_URL;`

Use `${url}/warehouse/` instead.

Reply:
Apply mo nalang din yung changes sa lahat ng /api